tox.ini

```diff
@@ -4,12 +4,13 @@ project = pre_commit_hooks
 envlist = py27,py33,py34,pypy,pypy3
 
 [testenv]
-deps = -rrequirements-dev.txt
+deps =
+    -rrequirements-dev.txt
 passenv = HOME HOMEPATH PROGRAMDATA
 commands =
     coverage erase
-    coverage run -m pytest {posargs:tests}
-    coverage report --show-missing --fail-under 100
+    coverage run --source=pre_commit_hooks/ -m pytest -rxs --durations 10 {posargs:tests}
```

Comment:
reporting coverage of tests is intentional -- it often exposes silly bugs about test naming or uncoverable useless test code